release notes/upcoming.md

```diff
@@ -8,6 +8,16 @@ You can now specify a file for all things logged by `console.log` to get written
 
 Thanks to @cheesedosa for both proposing and implementing this!
 
+### New result outputs: statsd and DataDog (#915)
+
+Both are very similar but DataDog has a concept of tags. By default both send on `localhost:8125` and currently only UDP is supported as transport.
+In order to change this you can use the `K6_DATADOG_ADDR` or `K6_STATSD_ADDR` env variable which has to be in the format of `address:port`.
+The new outputs also can add a `namespace` which a prefix before all the samples with `K6_DATADOG_NAMESPACE` or `K6_STATSD_NAMESPACE` respectively. By default the value is `k6.`, notice the dot at the end.
+In the case of DataDog there is an additional configuration `K6_DATADOG_TAG_WHITELIST` which by default is equal to `status, method`. This is a comma separated list of tags that should be sent to DataDog. All other tags that k6 emits are discarded. This is done because DataDog does indexing on top of the tags and some highly variable tags like `vu` and `iter` will lead to problems with the service.
```

Comment:
@robingustafsson Any opinion on this ? I am agnostic just want to note that people do complain that datadog gets with a lot of tags. We probably won't hit it given that we wouldn't have all that many but still

Reply:
I think we can start with this small set of whitelisted tags and then if it turns out users need more by default we can expand later. The only other tags that I would consider useful is the `url` and `name` that @na-- thinks should be excluded 😄. Given the kind of product that Datadog offers (being a long time customer) I don't believe anyone would really use it as a general purpose load testing result analysis tool. I can see though how you'd use it to store monitoring type of results but then you'd also likely set up custom metrics to track rather than  just the default metrics.

Reply:
@robingustafsson I don't think we should exclude both `url` and `name`, only `url`. Since they are either the same, or the user explicitly set `name` because `url` was too dynamic.
To reduce confusion, I think that the default value for `TagWhitelist` should be `proto, subproto, status, method, name, group, check, error, tls_version` (i.e. the `DefaultSystemTagList`, but without any highly variable things (and maybe with `error_code` instead of `error`, once that's merged?) )

Reply:
@na-- Ah ok, misread you. I'm not sure we need such an expansive tag whitelist though to start with for Datadog, but I'm not strongly opposed to it either 🙂Yes, `error_code` would be preferred over `error`, once available, if you ask me.

Reply:
Hmm something else that I realized - I'm not sure if a whitelist is actually the best approach here. It will work for system tags, but any custom user tags will be silently filtered, which isn't the best UX. If I, as a user, explicitly set a certain tag value on an HTTP request, I usually want it and can depend on it later for filtering the data...

In the InfluxDB collector, this is [handled](https://github.com/loadimpact/k6/blob/9af59bc1ca2eee2f065a544c9400c08875c5d268/stats/influxdb/config.go#L55) by transforming any highly variable tags into (non-indexable) fields. This is great, since user-supplied data won't be lost, while at the same time the InfluxDB server won't be overwhelmed with indexing highly-variable data.

I'm not very familiar with Datadog, but a quick check doesn't reveal any similar functionality there... So if no such functionality exists in Datadog, I wonder whether a blacklist (to block the most highly-variable tags like `vu`, `iter`,  `url`, and `error` (once `error_code` is merged)) will be more user-friendly than the current whitelist. Or whether we need such an option at all, considering that users can just set the emitted tags directly via the global `--system-tags` option?